userguide/services.rst

```diff
@@ -857,9 +857,11 @@ This section describes the configurable options for the
 Configure Rsyncd
 ~~~~~~~~~~~~~~~~
 
-:numref:`Figure %s <rsyncd_config_tab>`
-shows the rsyncd configuration screen which is accessed from
-:menuselection:`Services --> Rsync --> Configure`.
+To make adjustments to the
```

Comment:
This should really be more abstract, like it was. In other words, changing the service settings aren't necessarily limited to what's in rsyncd.conf, it could also change the database or other configuration files which are (and should be) opaque to the user.